R/zoop_synthesizer.R

```diff
@@ -303,8 +303,9 @@ Zoopsynther<-function(
     dplyr::distinct()%>%
     dplyr::left_join(dplyr::select(Crosswalk, "Taxname", "Level")%>%
                        dplyr::distinct(),
-                     by="Taxname",
-                     relationship = "many-to-one")%>%
+                     by="Taxname"#,
```

Comment:
Why was this change made?

Reply:
Ooo, I tried doing that to figure out why I was getting at error in the step, and I forgot to change it back. I'll change it back and rerun. Hopefully it still works!

Reply:
Of couse it didn't work. I found two more glitches in the crosswalk. Odonata is an order, not a family, Decapoda is an order, not a class. I wish I had better attention to detail. I think I'm getting closer...